vendor/brave-ios/tests/ledger_database_test.mm

```diff
@@ -656,24 +656,24 @@ - (void)testPublishersWithFiltersExcluded
   const auto filter = [[BATActivityInfoFilter alloc] init];
 
   // Filter all - skip exclude filtering
-  filter.excluded = BATExcludeFilterFilterAll;
+  filter.excluded = BATEXCLUDE_FILTERFilterAll;
```

Comment:
This naming is unfortunate 😅 I will have to fix the formatting for Obj-C generated enum names, or we can stop using all caps snake case for enums in mojo? It seems like it's a bit inconsistent atm: https://github.com/brave/brave-core/pull/2889/files#diff-99c488f19578bfd75399396011e2db1bR99

@NejcZdovc You have any preference?

Reply:
@kylehickinson I have spoken with @NejcZdovc and for now this is ok and we will raise a ticket for consistency

Reply:
Gotcha, I will push a mojo gen fix to this branch that refactors the all caps snake case then